scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/quartz/AppScalingScheduleJob.java

```diff
@@ -31,10 +30,7 @@ protected void executeInternal(JobExecutionContext context) throws JobExecutionE
 		Long scheduleId = dataMap.getLong("scheduleId");
 		Object schedule = dataMap.get("schedule");
 
-		logger.info("Job Key is " + jobKey);
-		logger.info("Application Id: " + appId + " Schedule Id: " + scheduleId);
-		logger.info("Schedule: " + schedule);
-		logger.info("==========================================");
-
+		logger.info(String.format("Job Key: %s, Application Id: %s, Schedule Id: %s, Schedule: %s", jobKey, appId,
```

Comment:
Sorry  didn't get, **should be combined with the above**

Is following ok?
logger.info("Job Key: " + jobKey + ", Application Id: " + appId + ", Schedule Id: " +scheduleId +", Schedule: " + schedule);